src/test/kotlin/com/exactpro/th2/common/schema/message/impl/rabbitmq/custom/TestMessageUtil.kt

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright 2021-2021 Exactpro (Exactpro Systems Limited)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.exactpro.th2.common.schema.message.impl.rabbitmq.custom
+import com.exactpro.th2.common.grpc.Direction.FIRST
+import com.exactpro.th2.common.grpc.Direction.SECOND
+import com.exactpro.th2.common.message.direction
+import com.exactpro.th2.common.message.get
+import com.exactpro.th2.common.message.message
+import com.exactpro.th2.common.message.messageType
+import com.exactpro.th2.common.message.sequence
+import com.exactpro.th2.common.message.sessionAlias
+import com.exactpro.th2.common.message.set
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Assertions.assertFalse
+import org.junit.jupiter.api.Assertions.assertNotEquals
+import org.junit.jupiter.api.Assertions.assertNotNull
+import org.junit.jupiter.api.Assertions.assertTrue
+import org.junit.jupiter.api.Test
+
+const val MESSAGE_TYPE_VALUE = "test message type"
+const val SESSION_ALIAS_VALUE = "test session alias"
+const val FIELD_NAME = "test field"
+const val FIELD_VALUE = "test value"
+val DIRECTION_VALUE = SECOND
```

Comment:
Might be private or internal. Currently, they are public and they will be available in other tests and might confuse users